format/view-spec.md

```diff
@@ -58,16 +55,16 @@ Writers create view metadata files optimistically, assuming that the current met
 
 The view version metadata file has the following fields:
 
-| Required/Optional | Field Name | Description |
-|-------------------|------------|-------------|
-| Required | format-version | An integer version number for the view format. Currently, this must be 1. Implementations must throw an exception if the view's version is higher than the supported version. |
-| Required | location | The view's base location. This is used to determine where to store manifest files and view metadata files. |
```

Comment:
I was reading the spec today and these 'manifest files' caught my attention. Then I wanted to see why it is missed in the review. Later found that there are open comments on the same. So, handled most of the comments.

Reply:
No need for "the". It is fine to say "where to store view metadata files".

Reply:
Updated.